src/test/java/teammates/test/automated/FeedbackSessionOpeningReminderTest.java

```diff
@@ -99,7 +99,8 @@ public void testAdditionOfTaskToTaskQueue() throws Exception {
         
         // Modify session to set emails as unsent but still open
         // by closing and opening the session. Also disable sending
-        // open emails.
+        // open emails, but in the current implementation open emails
```

Comment:
Would it be better if "current implementation" is removed from the comment? In future, if the implementation gets modified, but this comment isn't modified, then it will be really confusing for a future developer.